gradio/external.py

```diff
@@ -393,7 +310,7 @@ def query_huggingface_api(*params):
     return interface
 
 
-def get_spaces(space_name: str, api_key: str | None, alias: str, **kwargs) -> Blocks:
+def from_spaces(space_name: str, api_key: str | None, alias: str, **kwargs) -> Blocks:
```

Comment:
Unrelated to this change but if `iframe_url` is none we should raise an exception

Reply:
Ah yes we can do that